### PR TITLE
Office hours

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Migrate information regarding office hours that was previously here:
https://github.com/DataDog/dd-agent/wiki/Community-Office-Hours

Into the main documentation

### Motivation
Give more visibility to office hours

### Preview link

https://docs-staging.datadoghq.com/gus/office-hours/developers/office_hours/

### TO DO

* Add a desc
* Add a text in the dev section home page
* Add reference to this page on all open-source integration

### CODE REVIEW

File: content/developers/office_hours.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+**These sessions are held on the 2nd and 4th Wednesday of each month.**
+
+### Where
+
+Office hours will be held in #integrations in the [Datadog Community Slack][1] ([get an invitation][2]), in conjunction with a [Google Hangout][3].
+
+### When
+
+* 2nd Wednesday of each month at 10:00am eastern time (2:00pm UTC).
```
Comment (phrawzty) [inline content/developers/office_hours.md]: Thanks to the magic of daylight savings time, these correlations are not always true (EST is UTC-5, EDT is UTC-4). Since we aren't scheduling this according to UTC, the only safe way to express the schedule is:
> * 2nd Wednesday of each month at 10:00 AM Eastern Time.
> * 4th Wednesday of each month at 3:00 PM Eastern Time.

File: content/developers/office_hours.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+---
+title: Community Office Hours
+type: documentation
+---
+
+## Community Code Review
+
+Datadog runs twice monthly code reviews and office hours to assist community members who are contributing to our [open source projects](#list-of-open-source-project).  
+
+**These sessions are held on the 2nd and 4th Wednesday of each month.**
```
Comment (phrawzty) [inline content/developers/office_hours.md]: Why is this here, in bold, when it's stated just below in the *Where* section?

File: content/developers/office_hours.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+Office hours will be held in #integrations in the [Datadog Community Slack][1] ([get an invitation][2]), in conjunction with a [Google Hangout][3].
+
+### When
+
+* 2nd Wednesday of each month at 10:00am eastern time (2:00pm UTC).  
+* 4th Wednesday of each month at 3:00pm eastern time (7:00pm UTC).
+
+### Guidelines and Caveats
+
+- Review and feedback during office hours is offered on a first come first served basis. We may not get to every review each week. We appreciate your patience and understanding.
```
Comment (phrawzty) [inline content/developers/office_hours.md]: > … first come, first served …

A comma is used when writing this idiom.

File: content/developers/office_hours.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+- We welcome everyone from the community to join us and provide respectful and constructive feedback to their fellow members of the Datadog community. 
+
+- These sessions are not intended as a channel for general Datadog support. We are primarily focused on discussing and reviewing `datadog-agent` integrations, and contributions to other Datadog [open source projects](#list-of-open-source-project). For product support we encourage you to reach out to [our support team directly][4].
+
+- Sometimes a given PR maybe complex and require further offline review before we can provide feedback.
```
Comment (phrawzty) [inline content/developers/office_hours.md]: > may be

Two words, not one. 😄

File: content/developers/office_hours.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+- Review and feedback during office hours is offered on a first come first served basis. We may not get to every review each week. We appreciate your patience and understanding.
+
+- We welcome everyone from the community to join us and provide respectful and constructive feedback to their fellow members of the Datadog community. 
+
+- These sessions are not intended as a channel for general Datadog support. We are primarily focused on discussing and reviewing `datadog-agent` integrations, and contributions to other Datadog [open source projects](#list-of-open-source-project). For product support we encourage you to reach out to [our support team directly][4].
+
+- Sometimes a given PR maybe complex and require further offline review before we can provide feedback.  
+
+## List of open source project
```
Comment (phrawzty) [inline content/developers/office_hours.md]: Imho this list should be split out to a sub-section.

File: content/developers/office_hours.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+### Agent 
+
+| Repository                      | Description                                                                            | Status   |
+| :--------                       | :-------                                                                               | :------- |
+| [datadog-agent][]               | Datadog Agent Version 6                                                                | Active   |
```
Comment (phrawzty) [inline content/developers/office_hours.md]: `[datadog-agent][]` missing reference number

File: content/developers/office_hours.md (hunk 5 above)
Comment (l0k0ms) [reply content/developers/office_hours.md]: I tried with one list, and it was a little bit messy.. hence this sub-section division. But i'm totally open to change the layout if needed.

File: content/developers/office_hours.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+| Repository                     | Description                                         | Status   |
+| :--------                      | :-------                                            | :------- |
+| [documentation][19]            | The source for Datadog's documentation site         | Active   |
+| [deb-s3][35]                   | Easily create and manage an APT repository on S3    | Outdated |
+| [gohai][40]                    | System information collector                        | Active   |
+| [heroku-buildpack-datadog][41] | Heroku Buildpack to run the Datadog Agent in a Dyno | Active   |
+| [node-connect-datadog][44]     | Datadog middleware for Connect JS / Express         | Active   |
+
+
```
Comment (jeremy-lq) [inline content/developers/office_hours.md]: Let's remove some of the forks and outdated repos from this list.

Comment (l0k0ms) [reply content/developers/office_hours.md]: Should I remove all forked and outdated/deprecated repo?
